payload/attributes/interfaces.go

```diff
@@ -30,3 +32,11 @@ type ChainSpec interface {
 	EpochsPerHistoricalVector() uint64
 	SlotToEpoch(slot math.Slot) math.Epoch
 }
+
+// ReadOnlyBeaconState is the interface for a read-only beacon state.
+type ReadOnlyBeaconState interface {
+	GetSlot() (math.Slot, error)
+	ExpectedWithdrawals( /*timestamp*/ math.U64) (engineprimitives.Withdrawals, uint64, error)
```

Comment:
no harm doing this I believe
```suggestion
	ExpectedWithdrawals(timestamp math.U64) (engineprimitives.Withdrawals, uint64, error)
```

Reply:
Done